node/npm_dependencies.ts

```diff
@@ -233,24 +235,23 @@ export const vendorNPMSpecifiers = async ({
     return
   }
 
-  // To bundle an entire module and all its dependencies, create a barrel file
+  // To bundle an entire module and all its dependencies, create a entrypoint file
   // where we re-export everything from that specifier. We do this for every
   // specifier, and each of these files will become entry points to esbuild.
   const ops = await Promise.all(
-    npmSpecifiers.map(async ({ specifier, types }, index) => {
+    npmSpecifiers.map(async ({ specifier, types }) => {
       const code = `import * as mod from "${specifier}"; export default mod.default; export * from "${specifier}";`
-      const barrelName = `barrel-${index}.js`
-      const filePath = path.join(temporaryDirectory.path, barrelName)
+      const filePath = path.join(temporaryDirectory.path, `bundled-${slugifyPackageName(specifier)}.js`)
```

Comment:
I'm curious why you went with the `bundled-` prefix here, rather than just using the slugified package name?